lib/addons/aws-loadbalancer-controller/index.ts

```diff
@@ -74,6 +87,8 @@ export class AwsLoadBalancerControllerAddOn extends HelmAddOn {
             enableShield: this.options.enableShield,
             enableWaf: this.options.enableWaf,
             enableWafv2: this.options.enableWafv2,
+            createIngressClassResource: this.options.createIngressClassResource,
+            ingressClass: this.options.ingressClass
```

Comment:
Just curious what is the outcome if ingress class is specified but createIngressClassResource is false?

Reply:
If the createIngressClassResource is false the ingressClass does not get created and it is assumed that the user will manually create the ingressClass with the same name as the one defined in this.options.ingressClass (default = alb). This is the same behavior as the underlying Helm chart.